app/services/websocket.ts

```diff
@@ -159,5 +159,7 @@ export class WebsocketService extends Service {
     if (this.appService.state.argv.includes('--network-logging')) {
       electronLog.log(`WS: ${message}`);
     }
+
+    electronLog.log('WEBSOCKET', message, args);
```

Comment:
Hmm is this needed?  Seems incongruous with the `--network-logging` switch above.